docs/api/native-image.md

```diff
@@ -47,9 +52,9 @@ quality, it is recommended to include at least the following sizes in the:
   * 64x64 (200% DPI scale)
   * 256x256
 
-Check the _Size requirements_ section in [this article][icons].
+Check the _Size Requirements_ section in the [Windows Icon documentation][icons].
 
-[icons]: https://learn.microsoft.com/en-us/windows/win32/uxguide/vis-icons
+[icons]: https://learn.microsoft.com/en-us/windows/win32/uxguide/vis-icons#size-requirements
```

Comment:
I actually looked into these icons recently. I found the following link a bit more useful b/c:
- More recent and relevant to windows 10/11 vs other doc was created for windows 7
- Talks about sizes and scaling for different icons. More specific to creating icon files vs designing the icons. 
We don't have to change the link, just thought I'd share!

https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/app-icon-construction

Reply:
Awesome changes to the docs! I like how you made everything more clear and specific!

Reply:
I like the Icon Scaling section from the doc you found. I'll add it in! https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/app-icon-construction#icon-scaling